src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  &__star {
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    list-style: none;
+    background-image: url("../images/star.svg");
+  }
+}
+.stars--N .stars__star:nth-child(-n+4) {
```

Comment:
The class `stars--N` does not adhere to the BEM naming convention. In BEM, modifiers should be appended to the block or element with a double hyphen, and the base class should be a valid BEM block or element. Consider renaming this class to follow the BEM convention.